README.md

```diff
@@ -0,0 +1,3 @@
+# pizza_site
+
```

Comment:
hej @Onoma94 w PR powinny byc wszystkie Twoje zmiany - całe zadanie

Reply:
@lukasz-galka 
Przepraszam - za późno się zorientowałem, że powinienem był robić w osobnej gałęzi... w mainie jest ponad 20 commitów. Po prostu przez to, że cały projekt robiłem sam, na przemian dodając CSS i JS, to nie zwracałem uwagi na rozdzielanie na gałęzie.

Reply:
Mógłbym stworzyć nowe repozytorium i "przepisać" projekt odpowiednio rozgałęziając - może tak być?

Reply:
@Onoma94 zostaw juz jak jest